exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -1494,7 +1529,7 @@ func (b *Bitfinex) WsUpdateOrderbook(p currency.Pair, assetType asset.Item, book
 // GenerateDefaultSubscriptions Adds default subscriptions to websocket to be handled by ManageSubscriptions()
 func (b *Bitfinex) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription, error) {
 	var wsPairFormat = currency.PairFormat{Uppercase: true}
-	var channels = []string{wsBook, wsTrades, wsTicker, wsCandles}
+	var channels = []string{wsBook}
```

Comment:
Please reinstate the default subscriptions

Reply:
Whooooooa!
That's horrific. Sorry, really bad mistake. Would have happened during rebase.

Thanks for the catch 🙏